### ISSUE TITLE
Brigandine: The Legend of Runersia

### ISSUE THREAD

**Issue by EmulationFanatic** [Ryujinx/Ryujinx-Games-List#1851]
#### Current on `master` : 1.0.4761

Game loaded and was performing well but deadlocked about 10 minutes in.

#### Screenshots :
![2020-06-25 13_47_47-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789727-569eb480-b6ec-11ea-8a5a-25ded2f13513.png)
![2020-06-25 13_50_48-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789733-57374b00-b6ec-11ea-8e9a-e9116c51da74.png)
![2020-06-25 13_51_08-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789750-5d2d2c00-b6ec-11ea-9e05-0877458b1981.png)
![2020-06-25 13_51_31-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789760-61594980-b6ec-11ea-9a88-53ea95b4abda.png)
![2020-06-25 13_51_59-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789763-61f1e000-b6ec-11ea-8a4d-ef75cf2c6bd4.png)
![2020-06-25 13_52_21-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789781-6918ee00-b6ec-11ea-80fe-44fb6c91b4ab.png)
![2020-06-25 13_53_14-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789801-73d38300-b6ec-11ea-8404-90279142329b.png)
![2020-06-25 13_53_23-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789817-7c2bbe00-b6ec-11ea-82b1-d60c323d7701.png)
![2020-06-25 13_54_36-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789832-82ba3580-b6ec-11ea-9af9-48c3c4604689.png)
![2020-06-25 13_54_47-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789861-8b127080-b6ec-11ea-85ea-6b86652cdcad.png)
![2020-06-25 13_55_10-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789877-92d21500-b6ec-11ea-88a1-ef18c058d1ae.png)
![2020-06-25 13_56_06-Ryujinx 1 0 4761 - Brigandine_ The Legend of Runersia v1 0 0 (010011000EA7A000) ](https://user-images.githubusercontent.com/62343878/85789901-9d8caa00-b6ec-11ea-8ea5-7a3986da8a51.png)

#### Log file :

[BTLoR.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4833561/BTLoR.zip)

**Comment by Dissension** [comment]
can i test it?

**Comment by EmulationFanatic** [comment]
> can i test it?

Of course. Copy/paste the new issue template into a reply in this post and follow the formatting of other gamedb entries. Pending review, that test result will then become the original post in this thread.

**Comment by Dissension** [comment]
Thanks, but i'm newbee, so where i can download everything to get started?

**Comment by Dissension** [comment]
> > can i test it?
> 
> Of course. Copy/paste the new issue template into a reply in this post and follow the formatting of other gamedb entries. Pending review, that test result will then become the original post in this thread.

Thanks, but i'm newbee, so where i can download everything to get started?

**Comment by EmulationFanatic** [comment]
You can get Ryujinx from https://ryujinx.org/download. Then follow the guide at https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide-(Windows)

**Comment by Dissension** [comment]
> You can get Ryujinx from https://ryujinx.org/download. Then follow the guide at https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide-(Windows)

Thanks!

**Comment by Dissension** [comment]
Brigandine: The Legend of Runersia
Game Update Version : 1.0.2
Current on master : 1.0.4761
Game loaded and performing well, every mechanic available: quests, fights, class level, etc. Playable, average fps 20-30. Turns to black screen after alt+tab, save/load can help with it. Sounds are weird (maybe because of missing open al)

Hardware Specs :
CPU: Ryzen 7 2700X
GPU: NVIDIA GTX 2060 SUPER
RAM: 16GB
Screenshots :



![1](https://user-images.githubusercontent.com/59534007/86592802-cb9ba500-bf9c-11ea-9601-e64524ddf874.jpg)
![2](https://user-images.githubusercontent.com/59534007/86592812-cf2f2c00-bf9c-11ea-8f5f-3a016606c085.jpg)
![3](https://user-images.githubusercontent.com/59534007/86592819-d1918600-bf9c-11ea-8463-58535796e37c.jpg)
![4](https://user-images.githubusercontent.com/59534007/86592828-d48c7680-bf9c-11ea-9d30-9621e4a4f9f5.jpg)
Log file : 
[Brigandine.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4878849/Brigandine.zip)

**Comment by EmulationFanatic** [comment]
Updated. Thanks!

**Comment by Dissension** [comment]
> Updated. Thanks!

I need to update Firmware and open al, then if it's will be better i'll tell you)

**Comment by Chujok** [comment]
It seems to crash after some time in-game
I'm using ryujinx 1.0.4938
[Ryujinx_brigandine.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/4927417/Ryujinx_brigandine.log)

**Comment by madokaandhomura** [comment]
Game Update Version : 1.0.2
Current on master : 1.0.5289

After playing for 3 hours, the game's setting interface in the big map, organization and so on is reflected in 20fps. After entering the battle map, the selected unit instantly becomes 2fps, which is almost impossible to select. After a period of time, the combat map FPS can have 10 + FPS. After 3 hours, it will not be able to play normally. In the big map interface, it will become 2fps.
Hardware Specs :
CPU: Intel(R) Xeon(R) CPU E3-1230 V2 @ 3.30GHz
GPU: NVIDIA GTX 1080 
RAM: 16GB

- [ 
[Brigandine.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/5214041/Brigandine.log)
]

**Comment by EmulationFanatic** [comment]
Updated

**Comment by pemburubabi** [comment]
where to download the game? please help
